wiki/versions/1.19/getting_started/setup/markdown/setup.md

```diff
@@ -0,0 +1,101 @@
+# Setup
+
+Quilt is a mod-loader toolchain project for Minecraft; it allows you to modify and expand upon the game.
+
+It is recommended that you have knowledge of Java, because it is the language which Minecraft is written in. <!--- TODO: Perhaps link to some Java resources and courses. -->
+
+## Setting Up your Workspace
+
+While you could create your own template, cloning the [official template mod](https://github.com/QuiltMC/quilt-template-mod)
+might be a bit simpler.
+
+You can either clone the template with the [git command `clone`](https://git-scm.com/docs/git-clone), or download the
+ZIP file in the GitHub interface, and then unzipping it.
+
+### IDEs?
+
+Using an Integrated Development Environment is going to make your life easier while coding Minecraft mods, they provide
+several utilities and integrations for some tools, like Git, or Gradle.
+
+#### [IntelliJ IDEA](https://www.jetbrains.com/idea)
+
+Importing the mod with IntelliJ IDEA is as simple as selecting the folder/`build.gradle` file and opening it as a project.
+
+#### [Eclipse](https://www.eclipse.org)
+
+Open the project, and run the `eclipse` Gradle task.
+
+#### [Visual Studio Code](https://code.visualstudio.com)
+
+Visual Studio Code does not support Java by default, you must install an extension to be able to code in Java inside
+it.
+
+- [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack): A package of Java-related
+extensions for VSCode.
+
```

Comment:
Hmmm, I'm wondering if the VSCode extensions list should have the [Gradle extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-gradle) listed or not; On one hand, it may be useful, however, I remember it being quite intensive and buggy and such;
actually, perhaps ignore this

Reply:
It suggests that automatically iirc